count_zeros.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
```

Comment:
though we expected that you write your own function other than using imported ones.